src/pages/HomePage.css

```
@@ -1,1212 +1,1288 @@
-  width: 100%;
-  height: 100%;
+    width: 100%;
+    height: 100%;
 }
 
 .home-concept-map-bg path {
-  fill: none;
-  stroke: currentColor;
-  stroke-width: 1.1;
+    fill: none;
+    stroke: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)